src/ott/geometry/epsilon_scheduler.py

```diff
@@ -18,6 +18,8 @@
 
 __all__ = ["Epsilon"]
 
```

Comment:
I think if we add
```python
#: Some description of the constant below.
DEFAULT_SCALE = 0.05
```
and add it to the `.rst` file, we should be able to document it (either using :data: or :obj:).